app/dependencies.php

```diff
@@ -38,6 +38,7 @@
     $container['ProductModel'] = DI\factory('App\Factories\ProductModelFactory');
     $container['AddProductController'] = DI\factory('App\Factories\AddProductControllerFactory');
     $container['GetProductsController'] = DI\factory('App\Factories\GetProductsControllerFactory');
+    $container['GetProductsBySKUController'] = DI\factory('App\Factories\GetProductsBySKUControllerFactory');
```

Comment:
the path is incorrect as it is \GetProductBySKUControllerFactory, not Product(s). But please DIC index to not be Products as well.

Reply:
```suggestion
    $container['GetProductBySKUController'] = DI\factory('App\Factories\GetProductBySKUControllerFactory');
```